### PR TITLE
Update SoftLockListener.php

### PR DESCRIPTION
**Pull request by obernado**
The assignment is pointless

### CODE REVIEW

File: Listener/SoftLockListener.php [hunk 1]
```diff
@@ -17,7 +17,6 @@ public function __construct($maintenancePage, $maintenanceLock, array $whitePath
     {
         $this->maintenancePage = $maintenancePage;
         $this->lock = file_exists($maintenanceLock);
-        $this->whiteIps = $this->whiteIps;
```
Comment (wdalmut) [inline Listener/SoftLockListener.php:20]: Why this assignment is pointless? The `whiteIps` assignement comes from the services list

**Comment by obernado** [comment]
The left and right parts of the assignment are equal. Also later there is this assignment which assigns the correct value from the service: `$this->whiteIps = array_replace(array(), $whiteIps);`

File: Listener/SoftLockListener.php (hunk 1 above)
Comment (armellin) [inline Listener/SoftLockListener.php:20]: I guess that it should be:

`$this->whiteIps = $whiteIps;`

Hope this helps

**Comment by wdalmut** [comment]
Thanks for your PR merged with #7

**Pull request closed without merging**